README.md

```diff
@@ -5,7 +5,49 @@
 mgclient is a C library interface for [Memgraph](https://www.memgraph.com)
 database.
 
-# Building and installing on Linux
+## Building and installing on Apple
+
+To build and install mgclient from source you will need:
+   - CMake version >= 3.8
+   - OpenSSL version >= 1.0.2
+   - Apple LLVM/clang >= 8.0.0
+
+Once everything is in place, create a build directory inside the source
+directory and configure the build by running CMake from it:
+
+```
+mkdir build
+cd build
+cmake ..
+```
+
+If `cmake` can't locate OpenSSL, please set `OPENSSL_ROOT_DIR` to a valid path.
+An examples follows:
+
+```
+cmake -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1h ..
```

Comment:
What's up with this and why is this Apple specific?

Reply:
For some reason, CMake's `find_package` on macOS can't resolve include directories by default (freshly installed macOS with SSL on a default path). Setting `OPENSSL_ROOT_DIR` explicitly is the best "fix" I've found (all other alternatives are more specific, e.g., setting OpenSSL to include directories). I've included it here to write an example of what has to be done in case the CMake resolution fails. This whole part is Apple-specific, so I think it should be fine. Having an exact version of OpenSSL is probably the biggest problem in the whole thing.

Reply:
It's okay, was just wondering where the problem was. :+1: